modules/serverless-rn-1-35-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-35-0_{context}"]
+= Red Hat {ServerlessProductName} 1.35
+
+{ServerlessProductName} 1.35 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="new-features-1-35-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.15.
+* {ServerlessProductName} now uses Knative Eventing 1.15.
+* {ServerlessProductName} now uses Kourier 1.15.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.15.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.15.
+* The `kn func` CLI plugin now uses `func` 1.16.
+
+* Go functions using S2I builder are now available as a Generally Available (GA) feature for Linux and Mac developers, allowing them to implement and build Go functions on these platforms.
+
+* Workflows deployed using the `dev` or `preview` profiles are now automatically configured to generate monitoring metrics for Prometheus.
+
+* The Jobs Service supporting service can now be scaled to zero by configuring the `spec.services.jobService.podTemplate.replicas` field in the `SonataFlowPlatform` custom resource (CR).
+
+* Workflows deployed with the `preview` and `gitops` profiles are now automatically configured to send grouped events to the Data Index, optimizing event traffic.
+
+* A more comprehensive list of errors in the workflow definition is now provided, rather than only displaying the first detected error.
+
+* {ServerlessLogicProductName} is now certified for use with PostgreSQL version `15.9`.
+
+* Event performance between runtimes and the Data Index has been significantly enhanced through event batching. To enable event grouping, set the `kogito.events.grouping=true` property. For further optimization, you can use the `kogito.events.grouping.binary=true` property to activate an alternative serialization algorithm that reduces the size of grouped events. Additionally, enabling `kogito.events.grouping.compress=true` applies compression to grouped events, further reducing their size at the cost of slightly increased CPU usage.
+
+* Compensation states are now invoked when a workflow is aborted.
+
+* {ServerlessLogicProductName} now supports configuring the Knative Eventing system to produce and consume events for workflows and supporting services.
+
+* The secret configurations for the Broker and KafkaChannel (Apache Kafka) have been unified.
```

Comment:
@kaldesai sorry, this page is global to OpenShift serverless and not only to OpenShift Serverless Logic, right?

Reply:
f this is the case, shouldn't we somehow add a prefix {ServerlessLogicProductName} to all items related to us?

gmunozfe@redhat.com  dhanak@redhat.com zanini@redhat.com